src/blueapi/messaging/stomptemplate.py

```diff
@@ -180,12 +180,18 @@ def finished_connecting(_: Frame):
         self._listener.on_disconnected = self._on_disconnected
 
         LOGGER.info("Connecting...")
-        self._conn.connect(
-            username=self._authentication.username,
-            passcode=self._authentication.passcode,
-            wait=True,
-        )
-        connected.wait()
+
+        try:
+            self._conn.connect(
+                username=self._authentication.username,
+                passcode=self._authentication.passcode,
+                wait=True,
+            )
+            connected.wait()
+        except ConnectFailedException as ex:
+            LOGGER.error("Failed to connect")
```

Comment:
Mainly because logs can get a bit spammy it would be good to be specific here. Something like "Failed to connect to message bus at <IP:PORT>"